src/main/java/net/blay09/mods/waystones/block/WaystoneBlock.java

```diff
@@ -61,8 +73,8 @@ public WaystoneBlock() {
     }
 
     @Override
-    public VoxelShape getRenderShape(BlockState p_196247_1_, IBlockReader p_196247_2_, BlockPos p_196247_3_) {
-        return RENDER_SHAPE;
+    public VoxelShape getShape(BlockState state, IBlockReader world, BlockPos pos, ISelectionContext context) {
```

Comment:
`getShape` also affects the collision shape, but the collision doesn't have to be as complex as the selection shape (and probably the render shape doesn't need that either), so `getRenderShape` should probably be left as is and instead `getRaytraceShape` should be added for the selection box

Reply:
The collision shape and render shape are no more complex than a fence block used in a chain, and on top of that, Waystones are placed a lot less frequently than fences. Personally I think that simplifying the shapes here would be an unnecessary micro-optimization.